.github/workflows/ts_test.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
       - name: Start Oracle Server
         run: |
           yarn oracle-server &
+          ORACLE_SERVER_PID=$!
+          echo "ORACLE_SERVER_PID=$ORACLE_SERVER_PID" >> $GITHUB_ENV
```

Comment:
Cool trick, I didn't know:
1) & returns pid as exit code
2) You can catch exit code with $!
Learned a lot reading this PR :)